index.html

```diff
@@ -125,7 +125,7 @@
         <li target="https://support.mozilla.org/en-US/get-involved/kb" data-choice-id="sumoqa">Mozilla Knowledge Base
           <div class="extra" data-l10n-id="sumokb-extra"></div>
         </li>
-        <li target="https://support.mozilla.org/en-US/get-involved/army-of-awesome" data-choice-id="sumoaoa">Army of Awesome
+        <li target="https://wiki.mozilla.org/Army_of_Awesome" data-choice-id="sumoaoa">Army of Awesome
```

Comment:
According to https://blog.mozilla.org/sumo/2016/12/01/whats-up-with-sumo-1st-december/ we don't use "Army of Awesome" anymore, and we use "Social Support" now. Let's change this text to "Social Support" and link to https://wiki.mozilla.org/Support/Social instead.